src/main/java/dev/meyi/bn/modules/calc/BankCalculator.java

```diff
@@ -18,9 +18,30 @@
 
 public class BankCalculator {
 
+  public static double _purseLast = 0;
+  private static double _bazaarProfit = 0;
+  public static boolean _orderWait = false;
```

Comment:
again, these can't be public if prefaced by underscores - it goes against like every style guide ever. they shouldn't have underscores in front of them to begin with.